sdk/cosmosdb/cosmos/test/public/functional/NonStreamingQueryPolicy.spec.ts

```diff
@@ -333,7 +325,7 @@ describe("Vector search feature", async () => {
     });
   });
 
-  describe.skip("Vector Search Query large dataset", async () => {
+  describe("Vector Search Query large dataset", async () => {
```

Comment:
@amanrao23 we have purposefully skipped it. We don't want to execute large queries over our pipeline, this might impact our pipeline time.